### PR TITLE
feat: update chapter 6 of BG

### PR DESCRIPTION
**Pull request by Popespice**
## Summary
Updates to chapter 6 of the Beginner Guide.

### Location
docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md

Discord username (if different from GitHub): Popespice

### CODE REVIEW

**Review by Popespice — COMMENTED** [review]
Not sure why this thinks i touched Chapter 3, i made sure to leave it alone during editing.

File: docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md:1]: these will be overwritten by straks

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md [hunk 1]
```diff
@@ -6,7 +8,7 @@ from cruise altitude through STAR and Instrument Approach up to the final approa
 The actual final approach (ILS approach) will be covered by a separate chapter.
 
 !!! warning "Disclaimer"
-    The level of detail in this guide is meant to get a FlyByWire A320neo beginner safely from cruise level down to the ILS glideslope.
+    The level of detail in this guide is meant to get a FlyByWire A380X beginner safely from cruise level down to the ILS glideslope.
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]:
```suggestion
    The level of detail in this guide is meant to get a FlyByWire A380 beginner safely from cruise level down to the ILS glideslope.
```

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md [hunk 2]
```diff
@@ -32,13 +34,14 @@ This guide will cover these phases:
 
 ---
 
-## Base Knowledge: Route, Star and Approach
+## Base Knowledge: Route, Star, Approach and ILS
 
-- __Basics__<br/>
-  As we have learned in previous chapters, a flight route begins with an origin airport and a SID to safely guide the aircraft away from the airport to the first waypoint on their actual route. Similar to a SID, the arrival to an airport is done via a STAR (Standard Terminal Arrival Route) and an IAP (Instrument Approach, often APPR) to safely bring the aircraft into a position to land on the destination airport safely and with as little ATC communication as possible.
+??? tip "Basics"
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]: I would combine this into one collapsible  
Base Knowledge: Route, Star, Approach and ILS

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md [hunk 3]
```diff
@@ -83,24 +88,25 @@ This guide will cover these phases:
 **Situation:**
 
 - Aircraft is in `CRUISE` state and phase as per previous chapters.
-- Distance to destination is ~ 200 NM (on a short flight, start as early as possible and maybe even before the flight).
+- Distance to destination is ~ 200 NM (on a short flight, start as early as possible and maybe even before the flight).<p>
 
-#### Important Data Points for Descent Planning:
+!!! note "Important Data Points for Descent Planning:"
```
Comment (frankkopp) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]:
```suggestion
!!! tip "Important Data Points for Descent Planning:"
```

**Review by frankkopp — CHANGES_REQUESTED** [review]
Instead of the A32NX bg use the SOP as your template - e.g. you could even remove all the A320 stuff and build it up new (you can always copy stuff over)

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md
Comment (brocky120) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md:1]: Never used Github before so apologies if this wrong place etc but noticed a typo here:

tip "VNAV in the FlyByWire A830X"

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md [hunk 4]
```diff
@@ -247,62 +270,81 @@ A few minutes before we reach our calculated descent point (TOD) we request clea
...
+![FCU with ALT and V/S selector](../assets/beginner-guide/descent/FCU-AP-selector.png "FUC with ALT and V/S selector"){loading=lazy} <p style="color:yellow; font-size:18px;">TODO: update screenshot with new FCU</p>
 
 ATC typically will not clear us to our final target altitude directly, but will give us several step descents down to our required altitude.
 
+`DESCENT .......................................................... MONITOR`<br/>
+
 Also, ATC will often still expect us to respect the STAR's constraints, although they might have given us a lower clearance. We should then only descent to the constraint's altitude.
 
-!!! info "VNAV in the FlyByWire A32NX"
+??? tip "VNAV in the FlyByWire A830X"
```
Comment (2hwk) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]:
```suggestion
??? tip "VNAV in the FlyByWire A380X"
```

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: docs/pilots-corner/a380x/a380x-beginner-guide/.pages [hunk 5]
```diff
@@ -11,7 +11,7 @@ nav:
     - Preparing FMS: 03_preparing-fms.md
     - Engine Start and Taxi: 04_engine-start-taxi.md
     #- 05_takeoff-climb-cruise.md
-    #- 06_descent.md
+    - 06_descent.md: 06_descent.md
```
[the remark this answers is not in the document]
Comment (2hwk) [reply docs/pilots-corner/a380x/a380x-beginner-guide/.pages:14]: image

File: docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md [hunk 6]
```diff
@@ -515,7 +517,7 @@ Read the following tip for more information on FLEX temp.
 !!! tip "What is Flex Temp?"
     Flex temp is entered into the MCDU, enabling the computer to use the pilot-specified air temperature to allow for take-off thrust that is less than TOGA but not less than CLB. This is a method of creating cost savings by increasing engine life, resulting in reduced overhaul and fuel costs. This value is normally calculated via a pilot's company EFB or other tools.
 
-    Unfortunately, A320neo performance data for FLEX temp performance calculators alongside other various tools are not publicly available and are guarded by Airbus. For the purposes of simulation, it's important to note the following:
+    Unfortunately, A380 performance data for FLEX temp performance calculators alongside other various tools are not publicly available and are guarded by Airbus. For the purposes of simulation, it's important to note the following:
```
Comment (2hwk) [inline docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md]:
```suggestion
    Unfortunately, A380 performance data for FLEX temp performance calculators alongside other various tools are not publicly available and are guarded by Airbus. For the purposes of simulation, it's important to note the following:
```

Will have to check with don etc to see the status of this, might be different from the 20n

Comment (Popespice) [reply docs/pilots-corner/a380x/a380x-beginner-guide/03_preparing-fms.md]: Let me know and i can apply it to the guide.

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md [hunk 7]
```diff
@@ -48,26 +48,28 @@ This guide will cover these phases:
...
 
 !!! info "Microsoft Flight Simulator and navigation charts"
     Although Microsoft Flight Simulator allows some flight planning through the user interface, it is highly recommended to use navigational charts when flying airliners. MSFS' flight planning does not provide sufficient information to correctly fly a STAR or an approach.
 
-    There are several good sources for charts - free or subscription based. A good free solution is [ChartFox](https://chartfox.org/), which only requires a free VATSIM account. One of the most known subscription-based sources for charts is [Navigraph](https://navigraph.com/home). It is planned to integrate both solutions into the FlyByWire A32NX flyPad in the future.
+    There are several good sources for charts - free or subscription based. A good free solution is [ChartFox](https://chartfox.org/), which only requires a free VATSIM account. One of the most known subscription-based sources for charts is [Navigraph](https://navigraph.com/home). It is planned to integrate both solutions into the FlyByWire A380X flyPad in the future.
```
Comment (2hwk) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]:
```suggestion
    There are several good sources for charts - free or subscription based. A good free solution is [ChartFox](https://chartfox.org/), which only requires a free VATSIM account. One of the most known subscription-based sources for charts is [Navigraph](https://navigraph.com/home). It is planned to integrate both solutions into the FlyByWire A380X flyPad in the future.
```

Perhaps could be updated as navigraph has been available for a while. So only Chartfox is TBD

Comment (2hwk) [reply docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]: Also in A32NX bg

File: docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md [hunk 8]
```diff
@@ -173,12 +176,14 @@ ATC will quite often instruct certain altitudes within the STAR and APPR which d
 Before we descend, we should also reconfirm our destination runway and input or update our destination data into the MCDU PERF APPR page.
 This is typically done about 50 NM before the starting the descent.
 
-!!! info "Honeywell MCDU and PERF APPR Page"
+!!! info "<p style="color:yellow; font-size:18px;">TODO: update title with new MFD manufacturer</p>"
     It is important to note that when entering a value in the `TRANS ALT` field on this page that the flight crew reference the appropriate `TRANS LVL`. In this spec of the MCDU, Honeywell has not yet corrected the field to reflect `TRANS LVL` as the value required.
 
     As a quick reminder, when descending from cruise the flight crew is moving from a "Flight Level" to a local altimeter for arrival, which is where the `TRANS LVL` value is derived from.
```
Comment (2hwk) [inline docs/pilots-corner/a380x/a380x-beginner-guide/06_descent.md]: The second sentence of this may not apply (definitely no MCDU in the A380)

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Pull request merged**